openmetadata-service/src/main/java/org/openmetadata/service/apps/bundles/changeEvent/AbstractEventConsumer.java

```diff
@@ -157,7 +157,7 @@ private long loadInitialOffset(JobExecutionContext context) {
       return jobStoredOffset.getOffset();
     } else {
       EventSubscriptionOffset eventSubscriptionOffset =
-          getStartingOffset(eventSubscription.getId());
+          getStartingOffset(eventSubscription.getId(), 0);
```

Comment:
what is 0 indicates, is that the starting offset if there is none? wouldn't that cause send all the queued in events?

Reply:
oh it does not do anything , removing it , I was trying inlace changeEvent creation like search updates from repository to deprecate ChangeEventHandler, but will do this in some other PR